src/components/App.tsx

```diff
@@ -1936,6 +1937,10 @@ class App extends React.Component<AppProps, AppState> {
       return;
     }
 
+    if (this.state.penMode && isExcalidrawElement(this.state.elementType)) {
+      return;
+    }
```

Comment:
can we just disable on `gesture.pointers.size >= 2` (line 1936)?

Reply:
@dwelle: returning from `onGestureChange` if `gesture.pointers.size > 2 && penMode && isExcalidrawElement` results in a smooth drawing experience. If I try really hard, I can produce unwanted zoom and jump, but even deliberately trying it is hard... I need to slide my hand onto the screen before starting to draw, to achieve the jump. This way two-finger pinch-zoom and panning both works even in penMode!

Reply:
By my suggestion I meant that we already bail on `gesture.events.size === 2` a line above, so I think we can just make the check into `>= 2` and not check `penMode` or anything else.

Reply:
So you don't want 3 finger pinch zoom, not even in "selection" mode?

Reply:
We'll see, but right now it's not implemented anyway. And bear in mind that the `gesturechange` handler is for mac trackpad. Touchscreens are handled elsewhere.

Reply:
I made the change, but I can't test on a Mac touchpad.

Reply:
I tested on Mac and it works fine (at least there's no behavioral change from production).